Unix/share/networkschema/MSFT_LinkAggregationSettingData.mof

```diff
@@ -1,7 +1,7 @@
 // ===============================================================
-//  MSFT_LinkAggragationSettingData
+//  MSFT_LinkAggregationSettingData

```

Comment:
to match the filename.

fwiw, my tool has a mode to check file names, and I considered turning it on, in case i fixed a spelling in code but not of a file, but I was lazy and skipped that.